composer.json

```diff
@@ -16,23 +16,24 @@
         }
     ],
     "require": {
-        "php": "^5.3.3|^7.0",
+        "php": "^5.4|^7.0",
```

Comment:
Drop support for 5.3.3 since the build for 5.3 was removed. I didn't drop support for 5.x entirely since I don't know the desires of the maintainers.

Reply:
This is fine for now, the new branch will switch to 7.1+